packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -196,6 +196,34 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     }
   }
 
+  // Find next valid dropdown option
+  private _moveIndex(stepValue: number, index: number, selectedIndex: number): number {
+    // Return selectedIndex if nothing has changed
+    if (selectedIndex === index) {
+      return selectedIndex;
+    }
+
+    const { options } = this.props;
+    if (index < 0) {
+      // If index < 0, return first valid option
+      return this._moveIndex(stepValue, 0, selectedIndex);
+    } else if (index >= options.length) {
+      // If index > last option index, return last valid option
+      return this._moveIndex(stepValue, options.length - 1, selectedIndex);
+    } else {
+      // If current index is a header or divider, increment by step
+      while (options[index].itemType === DropdownMenuItemType.Header
+        || options[index].itemType === DropdownMenuItemType.Divider) {
+        // If index + stepValue is out of bounds, reverse step direction
+        if (index + stepValue < 0 || index + stepValue >= options.length) {
+          stepValue *= -1;
+        }
+        index = index + stepValue;
+      }
```

Comment:
Not entirely sure how to fix this, do you have a suggestion on how to take this into account?